test/unit/core.js

```diff
@@ -1523,26 +1523,33 @@ testIframe(
 );
 
 QUnit[ includesModule( "deferred" ) ? "test" : "skip" ]( "jQuery.readyException (original)", function( assert ) {
-	assert.expect( 1 );
+	assert.expect( 2 );
 
-	var message;
+	var done = assert.async();
+	var expected = "Error in jQuery ready #" + Math.random().toString( 36 ).slice( -7 );
+
+	this.sandbox.stub( jQuery, "readyException" ).callsFake( function( error ) {
+		assert.strictEqual( error.message, expected, "jQuery.readyException called with unexpected error" );
+		window.setTimeout( function() {
+			throw error;
+		} );
+	} );
 
 	this.sandbox.stub( window, "setTimeout" ).callsFake( function( fn ) {
+		var message;
```

Comment:
Hmm, i am now realizing this test still won't work properly if the tests are then run under older browsers, perhaps we could deprecate this `"jQuery.readyException (original)"` test given looks like `"jQuery.readyException (custom)"` will ultimately do the same.

Reply:
Well, did split them in the last commit, let me know if this is ok.

Btw, locally i got the below failed tests, but also over the `main` branch:

```
Chrome Headless 105.0.5195.102 (Windows 8.1) selector pseudo - has FAILED
Chrome Headless 105.0.5195.102 (Windows 8.1) selector pseudo - misc FAILED
```